baseline-refaster-rules/src/main/java/com/palantir/baseline/refaster/StreamEmpty.java

```diff
@@ -0,0 +1,21 @@
+package com.palantir.baseline.refaster;
+
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.AlsoNegation;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import java.util.Collection;
+import java.util.stream.Stream;
+
+public class StreamEmpty<T> {
+
+    @BeforeTemplate
+    Stream<T> streamNoParams() {
+        return Stream.of();
+    }
```

Comment:
I wonder if we have anything matching:
```java
Collections.emptySet().stream()
Collections.emptyList().stream()
ImmutableList.of().stream()
ImmutableSet.of().stream()
```

They might be _too_ silly

Reply:
That's pretty crazy

Reply:
Ya, I doubt we have them for real, let's ship this as is :-]